lisa/tools/hyperv.py

```diff
@@ -72,6 +72,7 @@ def create_vm(
         generation: int = 1,
         cores: int = 2,
         memory: int = 2048,
+        add_disks: bool = True,
```

Comment:
from the description, the default value should be false?

Reply:
The default value should be True in order to not break the existing callers of this function.

Reply:
ok, the parameter name could be like `attach_offline_disks`.

Reply:
Renamed.